sdk/storage/ci.yml

```diff
@@ -37,8 +37,3 @@ stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: storage
```

Comment:
Removing the artifacts stops the release pipelines being able to publish the package.